chpl/chpl-resources/src/main/resources-dev/jobs.xml

```diff
@@ -7,6 +7,9 @@
     <pre-processing-commands>
         <delete-triggers-in-group>triggerJob</delete-triggers-in-group>
         <delete-triggers-in-group>interruptJob</delete-triggers-in-group>
+        <!-- For OCD-2944 -->
+        <delete-triggers-in-group>*</delete-triggers-in-group>
```

Comment:
I don't like the `*` here. We don't need to delete the triggers that don't have relationships with ACBs, right? If we follow this path for PROD we'd need to recreate _every_ trigger, not just the ones that will break